### ISSUE TITLE
🚀[FEATURE]: Add ActionContext and ActionStatus to the public api

### ISSUE THREAD

**Issue by karlhaas** [ngxs/store#1637]
### Relevant Package
This feature request is for @ngxs/store

### Description
Please add ActionContext and ActionStatus to the public API. The import:
```
import { ActionContext, ActionStatus } from '@ngxs/store/src/actions-stream';
```
worked in Angular v8 but not v10.

### Describe the problem you are trying to solve
We have a special way to handle errors for which we observe the Actions stream an filter them manually. Therefore we use ActionContext and ActionStatus.

### Describe the solution you'd like
Add ActionContext and ActionStatus to the public api:

https://github.com/ngxs/store/blob/cae998d1dfcf1ef9e78813aaf6eda9687e5f9a62/packages/store/src/actions-stream.ts#L10-L22

**Comment by splincode** [comment]
@arturovt @markwhitfeld what do you think? i think we should do it

**Comment by arturovt** [comment]
Interesting one. @karlhaas would you provide a minimal example? P.S. just out of curiosity

**Comment by arturovt** [comment]
This has been fixed as part of the [v3.7.4](https://github.com/ngxs/store/releases/tag/v3.7.4) release.
Please test this out and let us know if this is not resolved.

**Comment by markwhitfeld** [comment]
Just a correction, this feature was only released in [v3.8.0](https://github.com/ngxs/store/releases/tag/v3.8.0).

Please leave a comment in the https://github.com/ngxs/store/discussions/1985 if you come across any regressions with respect to your issue.
